exercises/Cielo-Raymundo/answers.js

```diff
@@ -0,0 +1,123 @@
+// Question 1
+function greet(name) {
+  let answer=name;
+  return(`Good morning, ${answer}!`);
+}
+
+// Question 2
+function logOddNumbers() {
+  for(let i=1; i<100;i++){
+    if(i%2!=0){
+      console.log(i);
+    }
+  }
+}
+
+// Question 3
+function area(length, width) {
+  return length*width;
+}
+
+// Question 4
+function otherAngle(angle, angle2) {
+  let angle3;
```

Comment:
Great use of the check for negative numbers here but in the question it says you can assume positive numbers. Make this change so that the function body is only one line long.